frontend/web/src/app/components/sidemenu/sidemenu.component.html

```diff
@@ -14,30 +14,30 @@
       <li>
         <a routerLink="orders" class="nav-link" href="#" routerLinkActive="active">Orders</a>
       </li>
-      <li class="dropdown">
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Parts
-          <span class="caret"></span>
+      <li>
+        <a href="#parts" class="collapsed" data-toggle="collapse">Parts
+          <i class="glyphicon glyphicon-menu-down"></i>
         </a>
-        <ul class="dropdown-menu" role="menu">
+        <ul id="parts" class="sub-menu collapse" role="menu">
           <li>
             <a routerLink="digital-parts" routerLinkActive="active" href="#">Digital Parts</a>
           </li>
           <li>
-            <a routerLink="digital-parts" routerLinkActive="active" href="#">Physical Parts</a>
+            <a routerLink="physical-parts" routerLinkActive="active" href="#">Physical Parts</a>
           </li>
         </ul>
       </li>
 
-      <li class="dropdown">
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Prints
-          <span class="caret"></span>
+      <li>
+        <a href="#prints" class="collapsed" data-toggle="collapse">Prints
+          <i class="glyphicon glyphicon-menu-down"></i>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)